code/modules/awaymissions/corpse.dm

```diff
@@ -525,7 +525,7 @@
 	short_desc = "By unknown powers, your skeletal remains have been reanimated!"
 	flavour_text = "Walk this mortal plain and terrorize all living adventurers who dare cross your path."
 	assignedrole = "Skeleton"
-
```

Comment:
Instead of commenting it out, just add this line of code, which will give the skeleton an exile implant, rendering it unable to use the gateway. this'll also fix travis
/obj/effect/mob_spawn/human/skeleton/alive/equip(mob/living/carbon/human/H)
	var/obj/item/implant/exile/Implant = new/obj/item/implant/exile(H)
	Implant.implant(H)

Reply:
Ah, ya gotta indent those second and third lines or its not gonna work mate

Reply:
You put implant/implant(h) instead of implant.implant(h)